src/v1/RequirementsChecker.php

```diff
@@ -175,20 +221,20 @@ public function getErrors() {
 	 */
 	private function unmetRequirementDescription( $requirement ) {
 		// Requirement exists, but is out of date.
-		if ( ! empty( $requirement['exists'] ) ) {
+		if ( $this->parseRequirementProperty( $requirement, 'exists' ) ) {
```

Comment:
```suggestion
		if ( $this->parseRequirementProperty( $requirement, 'exists' ) && $this->parseRequirementProperty( $requirement, 'current' ) ) {
```

That way, if the current version returns `false` for any reason, it won't output "(you have )". Other than that, this is working nicely with MailPoet!